Tigran Kirakosov/style.css

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* Highier level containers states */
+.weather-container_main {
+  padding: 80px 110px;
+  width: 800px;
+  height: 600px;
+  margin: auto;
+  font-family: 'Dosis', sans-serif;
+  font-weight: normal;
+  background-image: url("/assets/images/bg/Tokyo-bg.png");
+  border-radius: 10px;
+  color: white;
+  cursor: default;
+}
+.weather-container_widget {
+  display: flex;
+  flex-flow: column wrap;
+  width: 580px;
+  height: 437px;
+  border-radius: 5px;
+  -webkit-box-shadow: 0px 0px 10px 1px rgba(0,0,0,0.75);
+  -moz-box-shadow: 0px 0px 10px 1px rgba(0,0,0,0.75);
+  box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.55);
+}
+/* First container state */
+.weather-container_first-half {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 100%;
+  width: 290px;
+  background-image: url("/assets/images/bg/Tokyo-bg-inner.png");
+  border-right: 1px solid rgba(51, 33, 38);
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+}
+.current-weather__state {
+  flex: 3;
+  padding-top: 43px;
+}
+.current-weather__state_image {
+  width: 142px;
+  height: 142px;
+}
+.current-weather__desc {
+  flex: 1;
+  padding-top: 24px;
+  font-family: 'Dosis', sans-serif;
+  font-weight: 200;
+  font-size: 36px;
+  background: rgba(53, 27, 36, 0.9);
+}
+.current-weather__date {
+  flex: 1;
+  padding-top: 40px;
+  font-family: 'Dosis', sans-serif;
+  font-weight: 300;
+  font-size: 22px;
+  background: rgba(119, 25, 33, 0.9);
+  border-bottom-left-radius: 5px;
+}
+/* Second container state */
+.weather-container_second-half {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  height: 100%;
+  width: 290px;
+  padding-bottom: 5px;
+  background: white;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+.current-weather__temperature {
+  font-family: 'Dosis', sans-serif;
+  font-weight: 200;
+  font-size: 101px;
+  color: #7b1f27;
+  border-bottom: 1px solid #7b1f27; 
+}
+.current-weather__graph {
+  border-bottom: 1px solid #7b1f27; 
+}
+.current-weather__graph_image {
+  position: relative;
+  bottom: 15px;
+}
+.current-weather__next-days {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 210px;
+  font-family: 'Dosis', sans-serif;
+  font-weight: 600;
+  font-size: 16px;
+}
+.day {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.current-weather__day {
+  border-bottom: 1px solid #7b1f27;

```

Comment:
I see that in general you use the good approach for ordering css properties - it's  cool :)  but try to not break it  - `border-bottom` shouldn't be before `width` if you are not sure about some cases - there is good example http://codeguide.academy/html-css.html

Reply:
Thank you :+1:
And thanks for the example - it's a very useful and informative tip!